python/ray/data/_internal/logical/planner.py

```diff
@@ -0,0 +1,34 @@
+from ray.data._internal.execution.interfaces import PhysicalOperator
+from ray.data._internal.logical.interfaces import LogicalOperator
+from ray.data._internal.logical.operators.read_operator import Read, plan_read_op
+from ray.data._internal.logical.operators.map_operator import (
+    MapBatches,
+    plan_map_batches_op,
+)
+
+
+class Planner:
```

Comment:
We may need a consistent usage of terms, I think planner is usually for generating optimized logical plan; and it's (physical) optimizer's job to generate physical plan?

Reply:
I was thinking that `Planner` here is like SparkSQL's `QueryPlanner`, which transforms a logical plan into a physical plan. https://github.com/apache/spark/blob/0c08289b9422e5fc9f57301be607fa272067ae7b/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/planning/QueryPlanner.scala#L55

Reply:
yeah I would love to stick with planner term here given above reason. Logical optimizer is to generate optimized logical plan.

Reply:
Ok, no particular opinion on these, it's fine as long as we are clear and consistent on the terms.